### ISSUE TITLE
viafree.se

### ISSUE THREAD

**Issue by Ljugis** [reek/anti-adblock-killer#1940]
http://www.viafree.se/program/humor/skolan/sasong-1/avsnitt-1
http://imgur.com/a/Vco9x
Google Chrome
Tampermonkey
Adblock + Adblock Plus
EasyList + AakList

**Comment by mapx-** [comment]
- remove adblock, keep only ABP
- add these filters
```
@@||viafree.se^$xmlhttprequest
@@||viafree.se^$generichide
```

**Comment by Ljugis** [comment]
Can't get it to work due to removing adblock and adding the two filters above to ABP

**Comment by mapx-** [comment]
it works fine for me in ABP (adding the filters above as custom filters)

**Comment by Ljugis** [comment]
"interesting".. It works when the filters are added in ABP but anti-adblock-killer turned off in Tampermonkey.
Well, the intended result is achieved, playing videos without commercials, thanks!

**Comment by flyingzebras** [comment]
https://github.com/reek/anti-adblock-killer/issues/1465

**Comment by mapx-** [comment]
Did you update easylist ?
see
https://forums.lanik.us/viewtopic.php?p=100758#p100758

**Comment by davidhedlund** [comment]
@mapx- 

> viafree detected AAK and blocked video streaming.

I suggest these labels for this issue:
- I had this problem without any blocker installed; it is completely related to AAK detection.
- Video
- Request

viafree.se block files:
Used in view-source:http://www.viafree.se/program/humor/frasier/sasong-8/avsnitt-1
- http://static.viafree.play.mtgx.tv/js/1.9.0-06f7417-20/player.min.js
- http://static.viafree.play.mtgx.tv/js/1.9.0-06f7417-20/client.min.js

viafree.se blocked functions:
- player.message.adblockerContent
- player.message.adblockerUsage

Also read https://github.com/reek/anti-adblock-killer/issues/1940#issuecomment-247251082

**Comment by davidhedlund** [comment]
https://github.com/EFForg/privacybadgerfirefox/issues/774

**Comment by davidhedlund** [comment]
@reek Meanwhile AAK Script don't have a solution for this I need to block it on viafree.se. However, viafree still block me when I add these lines AAK Script:
>// @exclude http://viafree.se/*
>// @exclude http://viaplay.tv/*
>// @exclude http://mtgx.tv/*

Should I add something more?

**Comment by flyingzebras** [comment]
@reek The solution is to add "viafree" to "var excludes". Can you please do this?

**Comment by davidhedlund** [comment]
[mylabel:important](https://github.com/search?utf8=%E2%9C%93&q=commenter%3Adavidhedlund+%22mylabel%3Aimportant%22+in%3Acomments&type=Issues&ref=searchresults)

**Comment by flyingzebras** [comment]
@reek The temporary solution for me was to add `"viafree.se",` in AAK Script to `var excludes` like this:

> ```
>   // issue: https://github.com/reek/anti-adblock-killer/issues/1940
>   var excludes = ["viafree.se", 
> ```

**Comment by thernztrom** [comment]
I can't get this to work. 
I use Chrome, tampermonkey, uBlock Origin, with Easylist and Anti-Adblock Killer by Reek.

I get the same message as in the first post, that it recognizes a adblocker. 

@flyingzebras I also tried adding this to the AAK script but didnt make any difference to me.
Am I doing something wrong, or have they changed something?

**Comment by flyingzebras** [comment]
@thernztrom Do you get this error in Firefox too?

**Comment by thernztrom** [comment]
I dont have Firefox installed :) @flyingzebras

**Comment by davidhedlund** [comment]
@thernztrom Then I cannot help, because I cannot install Chrome on my system as it allow verified free software only.

**Comment by WildS0ft** [comment]
@flyingzebras [solution](https://github.com/reek/anti-adblock-killer/issues/1940#issuecomment-251124095) works @thernztrom - If it doesn't it's because of another conflicting add-on/setting or you are not following the instructions correctly. I suggest you go through [this list](https://github.com/reek/anti-adblock-killer#help) and then follow the instructions here again.

I've tested this on three different computers with the Chrome browser and on a fourth computer with Firefox (Windows and Linux) using uBlock Origin.

**Comment by tooobe** [comment]
Any progress? Mine won't load or when I get it to load I get ads...

**Comment by flyingzebras** [comment]
@reek A friendly reminder:

Until a anti-ad blocker script for for viafree.se has been developed can you please add `"viafree.se",` in AAK Script to `var excludes` like this:

> ```
>   // issue: https://github.com/reek/anti-adblock-killer/issues/1940
>   var excludes = ["viafree.se", 
> ```

viafree.se cannot be used at all else.

**Comment by flyingzebras** [comment]
The trick to get rid of ads from viafree is right now to disconnect fwmrm.net and kill the script that demand that site to be connected.

**Comment by flyingzebras** [comment]
The trick to get rid of ads from viafree is right now to disconnect fwmrm.net and kill the script that demand that site to be connected.

**Comment by tooobe** [comment]
@flyingzebras Could you be a little more specific on how to do that? :) 👍

**Comment by flyingzebras** [comment]
@tooobe I'm sorry but I have no idea. I just noticed the Anti-ad message in a video when I blocked fwmrm.net to access it.

fwmrm.net goes to http://freewheel.tv/ and represent themselves "FreeWheel is the industry’s most complete advertising management solution." so they fiddle with "ad enforcement administration". =)

**Comment by tooobe** [comment]
@flyingzebras Aha I misunderstood your reply then, thought you had figured it out :)
Hopefully your hint will help someone figure it out then :)

**Comment by flyingzebras** [comment]
@tooobe Yeah. I don't know JS so I can't help much more. However, my findings are fresh so someone with JS skills should definitely have a look.

**Comment by flyingzebras** [comment]
https://github.com/reek/anti-adblock-killer/issues/2556

**Comment by flyingzebras** [comment]
@tooobe Stay tuned on https://github.com/reek/anti-adblock-killer/issues/3026
